policyengine_us/variables/gov/states/ca/ccrc/head_start/ca_cspp_eligible.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class ca_cspp_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "California State Preschool Program eligible"
+    definition_period = YEAR
+    reference = (
+        "https://www.ccrcca.org/headstart/programs/eligibility-requirements/"
+    )
+    defined_for = StateCode.CA
+
+    def formula(person, period, parameters):
+        ccrc_head_start_eligible = person(
+            "ca_ccrc_head_start_eligible", period
+        )
+        recieves_head_start = person("head_start", period)
+        return (recieves_head_start > 0) & ccrc_head_start_eligible
```

Comment:
What is a scenario where the person is head start eligible but does not receive the program?

Reply:
> What is a scenario where the person is head start eligible but does not receive the program?

yeah that's a good point, should I close this PR or instead point the variables like ca_cspp to head_start and ca_cctr to early_head_start

Reply:
i don't know if that would be double counting, because before this pr cspp and cctr aren't in the household state benefits so I'm not sure

Reply:
Do the rules state explicitly that you have to _receive_ head start just be eligible for it?